### DISCUSSION TITLE
Minor string quote error correction in script

### DISCUSSION THREAD

**Discussion by 20kaushik02** [SpotX-Official/SpotX#81] [Errors]
In line 1076, the path for test_spa isn't quoted. Either change it to use the xpui_spa_patch variable, or surround the path in double quotes.

Wasn't sure if this was enough for a PR, so here I am.

**Comment by amd64fox** [comment]
Thanks for being observant, in fact I just forgot to put the `$xpui_spa_patch` variable there, which is a little higher, it will be fixed with the next update.
